library/iFixit/Matryoshka/Memcached.php

```diff
@@ -60,10 +60,12 @@ public function decrement($key, $amount = 1, $expiration = 0) {
        $expiration);
    }
 
+   /** @return mixed|null */
```

Comment:
Can you use
```suggestion
   /** @return mixed|self::MISS */
```

Reply:
Cool! I didn't know that was possible: https://psalm.dev/r/a3bb06fde6